index.d.ts

```diff
@@ -0,0 +1,112 @@
+/** Declaration file generated by dts-gen */
+
+export = dog_statsy;
+
+declare class dog_statsy {
+    constructor(...args: any[]);
+
+    close(...args: any[]): void;
+
+    count(...args: any[]): void;
+
+    decr(...args: any[]): void;
+
+    flush(...args: any[]): void;
+
+    gauge(...args: any[]): void;
+
+    histogram(...args: any[]): void;
+
+    incr(...args: any[]): void;
+
+    meter(...args: any[]): void;
+
+    set(...args: any[]): void;
+
+    setFlushInterval(...args: any[]): void;
+
+    timer(...args: any[]): void;
+
+    trace(...args: any[]): void;
+
+    write(...args: any[]): void;
+
+    static captureRejectionSymbol: any;
+
+    static captureRejections: boolean;
+
+    static defaultMaxListeners: number;
+
+    static errorMonitor: any;
+
+    static init(opts: any): void;
+
+    static listenerCount(emitter: any, type: any): any;
+
+    static on(emitter: any, event: any): any;
+
+    static once(emitter: any, name: any): any;
+
+    static usingDomains: boolean;
```

Comment:
I don't we should add types for these properties because they're undocumented implementation details and people probably shouldn't be accessing them.